automount-actions/scripts/bin/auto-mount.sh

```diff
@@ -57,40 +57,29 @@ try_mount() {
 }
 
 # cleanup acts to ensure this daemon remains tidy
-cleanup () {
+cleanup() {
     _exit_status=$?
+
     sync
     
-    # Use find instead of ls for better reliability and portability
-    # This handles filenames with spaces and special characters properly
-    if [ -d "$MOUNT_DIR" ]; then
-        # Find all directories in MOUNT_DIR (these are our UUID mount points)
-        # Use -print0 and read -d '' to handle filenames with spaces/newlines
-        find "$MOUNT_DIR" -mindepth 1 -maxdepth 1 -type d -print0 | \
-        while IFS= read -r -d '' _mount_path; do
+    # Check if MOUNT_DIR exists and continue with our life
+    [ ! -e "$MOUNT_DIR" ] || {
+        for _mount_path in "$MOUNT_DIR"/*; do
```

Comment:
Nit: end quote at the last possible character.

```suggestion
        for _mount_path in "$mount_DIR/"*; do
```

Reply:
+1